amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -86,7 +86,7 @@ export class ProfilePage extends React.Component<ProfilePageProps> {
         <div className="container profile-page">
           <div className="row">
             <div className="col-xs-12 col-md-offset-1 col-md-10">
-              <Breadcrumb path='/' text='Search Results'/>
+              <Breadcrumb path="/" text="Home" />
```

Comment:
This one was fine to leave as `<Breadcrumb />`, but since we haven't exposed this in search yet it makes sense to leave it as directing to `Home`. Can you add a comment about removing the hardcoded path to `Home` when we are ready to expose this in production?